frontend/apps/marketing/src/components/section/Section.tsx

```diff
@@ -0,0 +1,17 @@
+import DSCOSection, {SectionProps} from '@code-dot-org/component-library/cms/section';
+
+// Import the background image used in the pattern backgrounds on Contentful
+import bgPatternImage from '@public/images/bg-pattern-lines.png';
+
+const Section = (props: SectionProps) => {
+  const {background} = props;
+
+  return <DSCOSection {...props} backgroundImageUrl={
+    background === 'patternDark' || background === 'patternPrimary'
+      ? bgPatternImage.src
+      : undefined
+    }
+  />;
+};
```

Comment:
nit:

```suggestion
const Section: React.FunctionComponent<SectionProps> = ({
  background,
  ...props
}) => (
  <DSCOSection
    {...props}
    backgroundImageUrl={
      background === 'patternDark' || background === 'patternPrimary'
        ? bgPatternImage.src
        : undefined
    }
  />
);
```

Reply:
Oh nice! I had to add the `background={background}` prop in there too, otherwise it was coming back `undefined` on every option.